### ISSUE TITLE
Region-state with Evil's visual block

### ISSUE THREAD

**Issue by ReneFroger** [xuchunyang/region-state.el#3]
Hi Chunyang! Thanks for sharing this package, I really appreciate it.

However, I have also noticed the region-state will not show any information in the minibuffer when you use the visual block from `evil` (you can try it with <kbd>C-v</kbd>).

**Comment by xuchunyang** [comment]
Yes, you're right, I can reproduce it.  Unfortunately, I don't use `evil` and don't know how to fix.

**Comment by ReneFroger** [comment]
Hmm, perhaps you could try a hook from `region-state` to `evil-visual-block`?
